src/arch/null/Kconfig

```diff
@@ -23,5 +23,5 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-Import('*')
-sticky_vars.Add(BoolVariable('USE_NULL_ISA', 'Enable NULL ISA support', False))
+config USE_NULL_ISA
```

Comment:
Does this work as expected? `NULL` should mean that you have *no* ISAs compiled. I.e., we should be able to build gem5 without any ISAs and it should work. I don't think that `NULL` should be a kconfig option.

I'll note that I tried enabling this and got an error that `AtomicSimpleCPU` doesn't exist (or something like that).

Reply:
Ok, I have deleted this file and change `USE_NULL_ISA` back to sticky var. The `USE_NULL_ISA` should not be configured by user.